test/e2e/bundle_e2e_test.go

```diff
@@ -47,6 +118,30 @@ var _ = Describe("Installing bundles with new object types", func() {
 				imageName  = "quay.io/olmtest/single-bundle-index:pdb"
 			)
 
+			// create VPA CRD on cluster
+			data, err := yaml.YAMLToJSON([]byte(vpaCRDManifest))
+			Expect(err).ToNot(HaveOccurred(), "could not convert vpa crd to json")
+
+			crd := &apiextensionsv1beta1.CustomResourceDefinition{}
+			err = json.Unmarshal(data, crd)
+			Expect(err).ToNot(HaveOccurred(), "could not convert vpa crd to unstructured")
+
+			_, err = kubeClient.ApiextensionsInterface().ApiextensionsV1beta1().CustomResourceDefinitions().Create(context.TODO(), crd, metav1.CreateOptions{})
+			if err != nil {
+				if !k8serrors.IsAlreadyExists(err) {
+					Expect(err).ToNot(HaveOccurred(), "could not create vpa crd to on cluster")
+				}
+			}
+
+			// ensure vpa crd is established and accepted on the cluster before continuing
+			Eventually(func () bool {
+				crd, err := kubeClient.ApiextensionsInterface().ApiextensionsV1beta1().CustomResourceDefinitions().Get(context.TODO(), crd.GetName(), metav1.GetOptions{})
+				if err != nil {
+					return false
+				}
+				return crdReady(&crd.Status)
+			}).Should(BeTrue())
```

Comment:
Returning the error as a second return value from the assertion will make any failure reports more useful, like this:
```suggestion
			Eventually(func () bool, error {
				crd, err := kubeClient.ApiextensionsInterface().ApiextensionsV1beta1().CustomResourceDefinitions().Get(context.TODO(), crd.GetName(), metav1.GetOptions{})
				if err != nil {
					return false, err
				}
				return crdReady(&crd.Status), nil
			}).Should(BeTrue())
```